enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotQuestion.stories.tsx

```diff
@@ -35,12 +33,296 @@ const Template: StoryFn<MetabotQuestionProps> = () => {
 
 export const Default = {
   render: Template,
+};
+
+export const RedirectReaction = {
+  render: Template,
+  parameters: {
+    msw: {
```

Comment:
mock the response the [official way](https://storybook.js.org/docs/writing-stories/mocking-data-and-modules/mocking-network-requests)